internal/app/machined/pkg/system/runner/process/process_test.go

```diff
@@ -223,6 +223,10 @@ func (suite *ProcessSuite) TestStopSigKill() {
 }
 
 func (suite *ProcessSuite) TestPriority() {
+	if os.Geteuid() != 0 {
+		suite.T().Skip("skipping test, need root privileges")
+	}
```

Comment:
Does it break w/o this?

Reply:
it breaks outside of container, requires more privs. I do smoke test via `go test ./...`

Reply:
hm, okay, maybe on different systems it's different